1

```diff
@@ -0,0 +1,7 @@
+Merge remote-tracking branch 'upstream/master'
+merging the switch case scenario removal 
+# Please enter a commit message to explain why this merge is necessary,
+# especially if it merges an updated upstream into a topic branch.
+#
+# Lines starting with '#' will be ignored, and an empty message aborts
+# the commit.
```

Comment:
What's this lel? Is it a dummy file? If it is please remove haha

Reply:
Yes i have removed them now :)